functions.sh

```diff
@@ -1013,7 +1013,7 @@ validate_vars() {
             graph_error "ERROR: CentOS 32bit doesn't support xfs on partition /"
             return 1
           fi
-          for j in `seq 1 $PART_COUNT`; do
+          for j in $(seq 1 $PART_COUNT); do
```

Comment:
Actually only the c-style loop is mentioned in the bash man page. We could also do a metric over all code on github comparing »seq« to »(())«.